apps/right-to-rent-check/behaviours/get-living-status.js

```diff
@@ -0,0 +1,18 @@
+'use strict';
+
+module.exports = superclass => class extends superclass {
+  locals(req, res) {
+    const locals = super.locals(req, res);
+    if (req.sessionModel.get('living-status') === 'yes') {
+      locals.livingStatusId = 'yes-living-status';
+      locals.livingStatusHeader = req.translate('pages.request-another-tenant.yes-living-status.header');
+      locals.livingStatusIntro = req.translate('pages.request-another-tenant.yes-living-status.intro');
+      return locals;
+    }
+    locals.livingStatusId = 'no-living-status';
+    locals.firstTenant = locals.tenants[0]['tenant-name'];
```

Comment:
This will break if there are no tenants, for instance, when a user deletes the last tenant?